lua52-sys/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub const LUA_ERRERR: c_int = 6;
 pub struct lua_State;
 
 pub type lua_CFunction = extern "C" fn(L: *mut lua_State) -> c_int;
+pub type lua_CFunction2 = unsafe extern "C" fn(*mut lua_State);
```

Comment:
Why not change the definition of `luaL_requiref` to use  `lua_CFunction` instead?